examples/create_webhook.rs

```diff
@@ -0,0 +1,26 @@
+use helius::types::{Cluster, WebhookType};
+use helius::{
+    types::{CreateWebhookRequest, TransactionType},
+    Helius,
+};
+
+#[tokio::main]
+async fn main() {
```

Comment:
Same comment re `main` that was made in `examples/append_address_to_webhook.rs`. All of the example files should follow this convention